src/tests/test_addresses.py

```diff
@@ -5,11 +5,27 @@
 from pybitmessage import addresses
 
 from .samples import sample_address, sample_ripe
+from struct import pack
 
 
 class TestAddresses(unittest.TestCase):
     """Test addresses manipulations"""
 
+    def test_encodeBase58(self):
+        """ Encode base alphabets and cover with 0 number """
+        self.assertEqual(
+            addresses.encodeBase58(0), "1", "Error in asserting")
+
+        self.assertEqual(
+            addresses.encodeBase58(1), "2", "Error in asserting")
+
+    def test_decodeBase58(self):
+        """ Decode a Base X encoded string """
+        self.assertEqual(
+            addresses.decodeBase58("0"), 0, "Error in asserting")
+        self.assertEqual(
+            addresses.decodeBase58("1"), 0, "Error in asserting")
```

Comment:
I covered it here: https://github.com/Bitmessage/PyBitmessage/pull/1788/files#diff-41b1875b15c45657de6a21377434a4352f6da8d40330451e4824dd16e3646905